packages/kit/src/core/adapt/utils.js

```diff
@@ -59,14 +60,13 @@ export function get_utils({ cwd, config, build_data, log }) {
 				const file = fs.readFileSync(target, { encoding: 'utf-8' });
 				const lines = file.split(/\r?\n/);
 				const new_lines = new Set(patterns);
-
 				// remove repeated lines
 				for (const line of lines) {
 					// this will prevent commented ignores to be reinserted
 					new_lines.delete(line.replace(/#\s*/, ''));
 				}
 				if (new_lines.size === 0) continue;
-				fs.writeFileSync(target, [...lines, ...new_lines].join('\n'));
+				fs.writeFileSync(target, [...lines, ...new_lines].join(EOL));
```

Comment:
should this detect what type of line endings are used in the file and preserve that?

Reply:
No, its the end of line of the platform the code is being executed on

Reply:
exactly, but it is possible that the native EOL is different from the style thats used in the file. sveltekit itself for example forces LF: https://github.com/sveltejs/kit/blob/f360a9ff4e3ac710d14edf5889829062658fad90/.editorconfig#L4

prefering the native EOL vs the existing eol in the file itself also leads to a bigger diff and may cause problems if checked in and then checked out on other platforms.